sklearn/svm/_classes.py

```diff
@@ -627,7 +629,8 @@ class SVC(BaseSVC):
 
     _impl = 'c_svc'
 
-    def __init__(self, C=1.0, kernel='rbf', degree=3, gamma='scale',
+    @_deprecate_positional_args
+    def __init__(self, *, C=1.0, kernel='rbf', degree=3, gamma='scale',
```

Comment:
I would leave `C` and `nu` positional, I think, but no strong preference.

Reply:
I think not. Too unclear with `SVC(0.5)`